src/logger.ts

```diff
@@ -1,37 +1,45 @@
 import { printHtml } from "kolmafia";
 
 const defaultHandlers = {
-  info: (message: string) => printHtml(`<b>[Libram]</b> ${message}`),
-  warning: (message: string) =>
+  info: (message: string): unknown => printHtml(`<b>[Libram]</b> ${message}`),
```

Comment:
I think we should have a unified function these all call instead of repeating the printHtml snippet across all functions

Reply:
Should people be able to modify the root printing function directly, or keep modifying the individual printing functions?